File: API/Entities/ToDoList.cs
namespace API_MySIRH.Entities
{
    public class ToDoList : EntityBase
    {
        public int Ordre { get; set; }
        public string Titre { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public ICollection<ToDoItem> ToDoItemList { get; set; } = null!;

    }
}

File: client/src/app/Models/TodoList.ts
import { todoItem } from "../Models/TodoItem";



export class todoList {
    Id:number=0;
    titre: string = "";
    description: string = "";
    creationDate: string ="";
    itemsNumber: number = 0;
    itemsNumerDone: number = 0;
    toDoItemList?: todoItem[];
}
